cla_public/templates/base.html

```diff
@@ -126,8 +126,10 @@
       fleeFromPage("mouse click");
     });
     $(document).keyup(function(e) {
-      if (e.keyCode == 27) { // escape key
-        fleeFromPage("escape key");
+      if ($('.laa-flee-button-enabled').length) {
```

Comment:
Given that we're only using one of the logic paths (ie there's no `else` behaviour), would this read more simply if combined into one `if <code> and <enabled>` statement rather than being nested?

Reply:
Yes, sure - I was thinking at the time that this way was a little more readable, but I guess it isn't really that different.